helm-tests/validate_typha_test.go

```diff
@@ -0,0 +1,106 @@
+package main_test
+
+import (
+	"testing"
+)
+
+// This file holds all of the functions for validating
+// expected values on the Typha deployment.
+
+func TestTypha(t *testing.T) {
```

Comment:
Grr, we use ginkgo/gomega everywhere else in Calico golang tests.. I think we need to be consistent. So, either switch these to use ginkgo/gomega... or switch all the other golang tests :D